inoculate/src/lib.rs

```diff
@@ -94,7 +94,7 @@ impl Options {
         &self,
         bootloader_manifest: &Path,
         kernel_manifest: &Path,
-    ) -> Result<PathBuf> {
+    ) -> Result<(PathBuf, PathBuf)> {
```

Comment:
nit/tioli: maybe worth turning this into a struct with named fields because it's not super obvious what the tuple of two pathbufs means? but, it's only used in one place so i don't care about this that much --- we can fix it later.